sign/dilithium/gen.go

```diff
@@ -59,6 +61,20 @@ func (m Mode) NIST() bool {
 	return strings.HasPrefix(m.Name, "ML-DSA-")
 }
 
+func (m Mode) OidGo() string {
```

Comment:
Just add a documentation pointer.

```suggestion
// https://csrc.nist.gov/Projects/computer-security-objects-register/algorithm-registration
func (m Mode) OidGo() string {
```